mautic/mautic
# Gated video fix for issue #3349

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | Y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #3349, #3177  
| BC breaks? | 
| Deprecations? | 

#### Description:
This PR fixes an issue where Gated videos did not render correctly if included in Dynamic Content. I have also attempted to fix another outstanding issue were other videos were rendered incorrectly.

Basically if a gated video is detected in the response for Dynamic Content, `MauticJS.initGatedVideo()` is called again to detect the new DOM content.

#### Steps to test this PR:
1. Run the steps outlined in #3349 and verify you see the behavior.
2. Install this PR and test again. This time you should see the video rendered correctly.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. See #3349
